src/sources/file_descriptors/mod.rs

```diff
@@ -192,3 +193,28 @@ async fn process_stream(
         }
     }
 }
+
+fn outputs(
```

Comment:
I would just drop a quick comment, something like

```suggestion

/// Builds the `vector_core::config::Outputs` for the respective `source_name` from the provided arguments.
fn outputs(
```

Reply:
Changed the text slightly--thanks!